src/backend/cgcv.h

```diff
@@ -15,13 +15,16 @@
 #define CGCV_H
 //#pragma once
 
+struct Symbol;
+struct Classsym;
```

Comment:
The types are used later in this file. Generally, I try to set things up so .h files don't have to have an order.